javascript/javascript3/week1/homework-js3-week1/giphy/giphy.js

```diff
@@ -0,0 +1,41 @@
+const inputSearch = document.getElementsByClassName('search')[0];
+const numberOfGifsInput = document.getElementsByClassName('number-of-gifs')[0];
+const button = document.getElementsByClassName('search-button')[0];
+const resultGifs = document.getElementsByClassName('gifs')[0];
+
+
+const getGifsUrl = () => {
+  const gifQuery = inputSearch.value;
+  let numberOfGifs = numberOfGifsInput.value;
+    if (!gifQuery) {
+        resultGifs.innerHTML = '';
+        const errorText = document.createElement('h1')
+        errorText.textContent = 'please enter search value'
+        resultGifs.append(errorText)
+        return
+    } if (!numberOfGifs) {
+        numberOfGifs = 100
+    }
+  fetch(
+    `https://api.giphy.com/v1/gifs/search?api_key=HGNr7Qx2kRcKKwRg4A1vxGtS7cUgzp9a&q=${gifQuery}&limit=${numberOfGifs}&offset=0&rating=pg-13&lang=en`
```

Comment:
To avoid having long lines like this, you can create a variable with the base url and add to it. Even better - make a function that will give you the query result based on the inputs you supply.